data.go

```diff
@@ -10,12 +10,13 @@ import (
 
 // TrianglesData specifies a list of Triangles vertices with three common properties:
 // TrianglesPosition, TrianglesColor and TrianglesPicture.
-type TrianglesData []struct {
-	Position  Vec
-	Color     RGBA
-	Picture   Vec
-	Intensity float64
+type vertex struct {
+	Position   Vec
+	Color	   RGBA
+	Picture	   Vec
+	Intensity  float64
 }
+type TrianglesData []vertex
```

Comment:
I don't think this is a good idea. First of all, the unexported `vertex`, at least it should be exported. But even then, I think it would be a redundant type. AFAIK, you only use this type when appending to `TrianglesData`, right? For that one usecase, you can use an anonymous struct literal instead.